src/ORM/LazyEagerLoader.php

```diff
@@ -140,6 +140,7 @@ protected function _injectResults(iterable $objects, $results, array $associatio
         $injected = [];
         $properties = $this->_getPropertyMap($source, $associations);
         $primaryKey = (array)$source->getPrimaryKey();
+        /** @param \Cake\Collection\CollectionInterface */
```

Comment:
variable missing for param?

Reply:
It should be `@var`

Reply:
Unless this is something for a closure or inline method. `function ($e) use ($primaryKey)` sure looks like it.

Reply:
`$e` looks like Entity, should be typehinted or have better varname